src/components/OptionsList/types.ts

```diff
@@ -10,7 +10,7 @@ type Section = {
     title: string;
 
     /** The initial index of this section given the total number of options in each section's data array */
-    indexOffset: number;
+    indexOffset?: number;
```

Comment:
Why are we making this optional instead of removing it, or explicitly passing it everywhere?